digicampipe/image/cones_image.py

```diff
@@ -0,0 +1,705 @@
+from digicampipe.utils import geometry
+from digicampipe.image.kernels import *
+from digicampipe.image.utils import *
+from cts_core.camera import Camera
+from astropy import units as u
+from matplotlib.patches import Circle, Arrow
+import numpy as np
+from decimal import *
+import os
+
+
+class ConesImage(object):
+    def __init__(self, image, image_cone=None, output_dir=None,
+                 digicam_config_file='./tests/resources/camera_config.cfg'):
+        """
+        constructor of a ConesImage object.
+        :param image: fit filename or numpy array containing the lid CCD image.
+        If set to 'test', a test image is created.
```

Comment:
image plays a triple role: array, path-string pointing to a fits file, or the special string `test`. This is quite difficult to understand and maintain.

Reply:
Would it be better to create a ConeImageSim class inheriting from ConeImage to handle the "test" case instead ?

Reply:
No, I don't think so. I'll make a proposal in a new PR .. gimme a second.